### PR TITLE
Add Stage1Aot to size regression

### PR DESCRIPTION
**Pull request by eerhardt**
This will auto file issues when the Stage1Aot size regresses.

@sebastienros - is this all that needs to be done? How can I test these changes?

### CODE REVIEW

File: build/regressions.config.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ sources:
         - area-perf
       owners:
         - sebastienros
+        - eerhardt
```
Comment (sebastienros) [inline build/regressions.config.yml:88]: You can add yourself only for Stage1Aot (same for labels)

```yaml
    rules:
      - include: FortunesEF
      - include: Stage1Aot
        owners:
          - eerhardt
```

Comment (sebastienros) [reply build/regressions.config.yml:88]: Does this sample count as documentation?

https://github.com/dotnet/crank/blob/main/src/Microsoft.Crank.RegressionBot/sample.config.yml#L7-L14

Comment (eerhardt) [reply build/regressions.config.yml]: Done.

**Comment by eerhardt** [comment]
Ok, does this look better?

Any thought to removing `daysToSkip`? For size, the changes don't swing too much. I don't think we need 2 days before logging the issue.

**Pull request merged**